### PR TITLE
GEOMESA-2015 Not registering coprocessors if remote filtering is disabled

### PR DESCRIPTION
**Pull request by elahrvivaz**
Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

File: geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/index/HBaseFeatureIndex.scala [hunk 1]
```diff
@@ -92,16 +90,12 @@ trait HBaseFeatureIndex extends HBaseFeatureIndexType
         logger.debug(s"Creating table $name")
         val descriptor = new HTableDescriptor(name)
         descriptor.addFamily(HBaseFeatureIndex.DataColumnFamilyDescriptor)
-        Option(admin.getConfiguration.get(CoprocessorHost.USER_REGION_COPROCESSOR_CONF_KEY)) match {
+        if (ds.config.remoteFilter) {
```
Comment (jnh5y) [inline geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/index/HBaseFeatureIndex.scala]: What's the default for remote filtering?  I'm wondering if the command line tools will default to setting up the coprocessors or not...

**Comment by jnh5y** [comment]
@aheyne thoughts?

**Comment by aheyne** [comment]
Makes sense to me.

File: geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/index/HBaseFeatureIndex.scala (hunk 1 above)
Comment (elahrvivaz) [reply geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/index/HBaseFeatureIndex.scala]: default is true. it checks the ds param first, then the system prop, then defaults to true

**Pull request merged**